packages/react/src/river/RiverAccordion/RiverAccordion.module.css

```diff
@@ -0,0 +1,161 @@
+.RiverAccordion {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto;
+  grid-template-areas:
+    'visual'
+    'accordion';
+  gap: 0;
+  position: relative;
+}
+
+@media (min-width: 48rem) {
+  .RiverAccordion {
+    grid-template-rows: auto;
+    gap: 8.33%; /* 100% ÷ 12 = 8.33%, or 1 grid unit */
+  }
+
+  .RiverAccordion__align-start {
+    grid-template-columns: 4fr 7fr; /* 12 columns, minus 1 for the gap */
+    grid-template-areas: 'accordion visual';
+  }
+
+  .RiverAccordion__align-end {
+    grid-template-columns: 7fr 4fr; /* 12 columns, minus 1 for the gap */
+    grid-template-areas: 'visual accordion';
+  }
+}
```

Comment:
The math for the grid does not seem to align properly with the columns of the `Grid`. We assume there are 12 fractions of the available space, but we are not considering the gaps between the columns.

The gap between the visual and the accordion should equal 1 Grid column plus 1 gap. We can use `var(--brand-Grid-spacing-column-gap)` to accurately calculate the final spacing.

Reply:
Great spot @danielguillan, thank you! 🙏

I've corrected that now and double-checked the maths and the rendered sizes, but take a look and let me know what you think.